lib/firebase/repository/user_repository.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../model/user/user.dart';
+import '../firestore_refs.dart';
+
+final userRepositoryProvider = Provider.autoDispose((ref) => UserRepository());
+
+class UserRepository {
+  Future<void> addUserId() async {
```

Comment:
```suggestion
  Future<void> createUser() async {
```

UserIdを追加するというより,Userを追加する方が適切なので `createUser` `addUser`とかの方がbetterそう